lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -1155,12 +1162,12 @@ class MongoClientInterface {
             ops.push(masterOp);
             return c.bulkWrite(ops, {
                 ordered: true,
-            }).then(() => cb(null, `{"versionId": "${objVal.versionId}"}`)).catch((err) => {
+            }).then(() => cb(null, `{"versionId": "${escapedVersionID}"}`)).catch((err) => {
```

Comment:
In those contexts, I'd prefer for ease of maintenance to do directly `JSON.stringify({ versionId: objVal.versionId })` and let the JSON stringifier escape accordingly.